packages/integrations/node/README.md

```diff
@@ -150,6 +150,17 @@ You can pass the path to your key and certification via the environment variable
 SERVER_KEY_PATH=./private/key.pem SERVER_CERT_PATH=./private/cert.pem node ./dist/server/entry.mjs
 ```
 
+#### Runtime environment variables
+
+If an `.env` file containing environment variables is present when the build process is run, these values will be hard-coded in the output, just as when generating a static website.
+
+During the build, the variable must be absent from the `.env` file, and you must provide Astro with every environment variable to expect at run-time: `VARIABLE_1=placeholder astro build`. This signals to Astro that the actual value will be available when the built application is run. The placeholder value will be ignored by the build process, and Astro will use the value provided at run-time.
```

Comment:
Thank you for your input @TheOtterlord, this looks very nice! Before merging it, though, do you think it would make sense to make this clearer? "During the build, the variable must be absent from the `.env` file" does not say which variable. Could it be "the runtime variables"

Reply:
Yep! That sounds like a great clarification

Reply:
I’ve adjusted it like this. How does the PR look to you now?